user-interface/src/case-detail/panels/CaseDetailHeader.tsx

```diff
@@ -35,6 +34,22 @@ export default function CaseDetailHeader(props: CaseDetailHeaderProps) {
     }
   };
 
+  function printCaseIdHeader() {
+    return (
+      <h2 className="case-number text-no-wrap" title="Case Number">
+        {props.caseId}{' '}
+        <button
+          id="header-case-id"
+          className="usa-button usa-button--unstyled"
+          onClick={() => copyCaseNumber(props.caseId)}
+          title="Copy Case Id to clipboard"
+        >
+          <Icon name={'content_copy'}></Icon>
+        </button>
```

Comment:
I wonder if we will eventually want a new component for buttons that are icons so we don't have the USWDS class names in our tsx. Not super important now.

Reply:
Good point and .. I thought perhaps we already had one.  I could probably throw one together real quick... maybe :-)